src/frontend/src/components/Footer/index.tsx

```diff
@@ -4,9 +4,15 @@ export default class Footer extends React.Component {
   public render() {
     return (
       <footer className="mw8 center bg-white pt5 pb6 ph4">
-        <a className="link underline hover-blue" href="/manual#privacypolicy">
+        <a className="link hover-blue underline pr5" href="/privacy-policy">
           Privacy Policy
         </a>
+        <a className="link hover-blue underline pr5 pl4" href="/faq">
+          FAQ
+        </a>
+        <a className="link hover-blue underline pl4" href="/appendix">
+          Appendix
+        </a>
```

Comment:
The footer gets a bit tangled on skinnier screens:
![Screen Shot 2020-06-25 at 8 09 52 PM](https://user-images.githubusercontent.com/6887982/85810431-a4cdab00-b720-11ea-8789-973c581ae4ea.png)

```suggestion
        <a className="dib link hover-blue bb mr5 mb4" href="/privacy-policy">
          Privacy Policy
        </a>
        <a className="dib link hover-blue bb mr5 mb4" href="/faq">
          FAQ
        </a>
        <a className="dib link hover-blue bb" href="/appendix">
          Appendix
        </a>
```